test_vm/src/lib.rs

```diff
@@ -0,0 +1,90 @@
+use cid::Cid;
+use fvm_ipld_hamt::{BytesKey, Hamt, Sha256};
+use fvm_shared::address::Address;
+use fvm_shared::bigint::bigint_ser;
+use fvm_shared::blockstore::MemoryBlockstore;
+use fvm_shared::econ::TokenAmount;
+use fvm_shared::encoding::tuple::*;
+use std::error::Error;
+use std::fmt;
+
+pub struct VM<'bs> {
+    store: &'bs MemoryBlockstore,
+    state_root: Cid,
+    actors_dirty: bool,
+    actors: Hamt<&'bs MemoryBlockstore, Actor, BytesKey>,
+}
+
+impl<'bs> VM<'bs> {
+    pub fn new(store: &'bs MemoryBlockstore) -> VM<'bs> {
+        let mut actors = Hamt::<&'bs MemoryBlockstore, Actor, BytesKey, Sha256>::new(store);
+        VM { store, state_root: actors.flush().unwrap(), actors_dirty: false, actors }
+    }
+
+    pub fn get_actor(&self, addr: Address) -> Result<Actor, TestVMError> {
```

Comment:
I'm still getting a feel for this myself, but taking the `Runtime` interface as example, `Address` should probably be passed by reference here (which will be propagated to `get()`) and elsewhere.